src/xrc/src/api.rs

```diff
@@ -736,15 +767,12 @@ fn handle_fiat_pair(
 }
 
 async fn get_stablecoin_rate(
+    exchanges: &[&Exchange],
     symbol: &str,
     timestamp: u64,
-) -> Result<QueriedExchangeRate, CallExchangeError> {
+) -> Result<QueriedExchangeRateWithFailedExchanges, CallExchangeError> {
     let mut futures = vec![];
-    EXCHANGES.iter().for_each(|exchange| {
-        if !cfg!(feature = "ipv4-support") && !exchange.supports_ipv6() {
-            return;
-        }
-
+    exchanges.iter().for_each(|exchange| {
```

Comment:
Why has the feature flag check been removed?

Reply:
After talking with @dfinity-ryancroote , we decided that this is essentially `exchange.is_available()`, so this call is redundant.